projects/postgresql/add_fuzzers.diff

```diff
@@ -0,0 +1,37 @@
+diff --git a/src/backend/Makefile b/src/backend/Makefile
```

Comment:
This change needs to go upstream, is developer not allowing this, we can wait till this gets checked in.